application/helpers/invoice_helper.php

```diff
@@ -104,3 +104,28 @@ function invoice_recMod10($in)
 
     return (10 - $carry) % 10;
 }
+
+function currencies()
```

Comment:
Why only 3 currencies are available? Plus, "Dollar" is too generic, USD should be used instead